trinity/utils/chains.py

```diff
@@ -114,17 +122,32 @@ class ChainConfig:
 
     port: int = None
 
+    bootstrap_nodes: Tuple[Node, ...] = None
+
     def __init__(self,
                  network_id: int,
                  data_dir: str=None,
                  nodekey_path: str=None,
                  nodekey: PrivateKey=None,
                  sync_mode: str=SYNC_FULL,
-                 port: int=30303) -> None:
+                 port: int=30303,
```

Comment:
Total sidetrack: this line change makes me wonder if we can improve the style guide for long function signatures. Something like:

```py
def a(
    b,
    c,
    d,
    e,
) -> ReturnType:
    do_something()
```

Reply:
I generally agree.  I've not found a solution I like for this and type hints are still something that I'm getting used to (and that we need to add to the style guide).